doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -2,6 +2,9 @@
 
 Everyone is welcome to help improve the content of the Git and Github training resources. There are two repositories linked to the Git and Github project i.e. [learning resource repo](https://github.com/Ifycode/git-github-training-resource) and the [training repo](https://github.com/Ifycode/git-github-training). Interested contributors can follow the guides below when contributing to any of the Git and Github training repositories.
 
+{% hint style="info" %}
+ Interested contributors should note that taking the [Git and Github training](https://obiagba-mary.gitbook.io/git-and-github-training) can help you get better at contributing to the Git and Github training project (and any other Github projects they will be taking on in the future).
+{% endhint %}
```

Comment:
Nice job formatting as a hint @oyenuga17

Help make the following changes:
- Add a space in between the hint and the section after it. They are too close.
- Your pull request title let it begin with `Fix: `

Reply:
i have made changes at @Ifycode

Reply:
Thank you 👍💯